avm/res/web/serverfarm/main.bicep

```diff
@@ -110,7 +110,7 @@ resource avmTelemetry 'Microsoft.Resources/deployments@2023-07-01' = if (enableT
 resource appServicePlan 'Microsoft.Web/serverfarms@2022-09-01' = {
   name: name
   kind: kind
-  location: location
+  location: 'eastus'
```

Comment:
Is it required to be hardcoded in the template?

Reply:
After discussion with Alex it is due to the parameter being over written in the pipeline - this current change just needs to be to be a per module change and therefore must be hardcoded to the location value

Reply:
Updated! fixed `main.bicep`